src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -100,6 +127,24 @@ def dcos_browse(name, resource_group_name, disable_browser=False):
 
     return
 
+def acs_install_cli(resource_group, name, install_location=None, client_version=None):
+    acs_info = _get_acs_info(name, resource_group)
+    orchestrator_type = acs_info.orchestrator_profile.orchestrator_type  # pylint: disable=no-member
+    if  orchestrator_type == 'kubernetes':
+        # TODO: is there a more python idiomatic way to do this?
+        if client_version:
```

Comment:
Not if you come in the `acs install-cli ...` path, since we can't have a default in there, since it's orchestrator dependent, and we don't know which it is 'til we do the lookup.